frontend/projects/ui/src/app/components/backup-drives/backup-drives.component.html

```diff
@@ -15,27 +18,50 @@
   <ng-template #noError>
     <ion-item-group>
       <!-- ** cifs ** -->
-      <ion-item-divider>Shared Network Folders</ion-item-divider>
+      <ion-item-divider>LAN Shared Folders</ion-item-divider>
       <ion-item>
         <ion-label>
           <h2>
-            Shared folders are the recommended way to create Embassy backups. View the <a href="https://docs.start9.com/user-manual/general/backups.html#shared-network-folder" target="_blank" noreferrer>Instructions</a>
+            LAN shared folders are the recommended way to create Embassy
+            backups. View the
+            <a
+              href="https://docs.start9.com/user-manual/general/backups.html#lan-backups"
```

Comment:
It is my understanding that the `docs.` prefix will not be required.  However, due to versioning, you will require start9.com/latest/.....  
Also with the restructure, the final link should be https://start9.com/latest/user-manual/backups/backup-create/#lan-backups